crates/viewer/re_renderer/src/video/mod.rs

```diff
@@ -18,7 +18,7 @@ pub enum VideoPlayerError {
     #[error("The decoder is lagging behind")]
     EmptyBuffer,
 
-    #[error("Video seems to be empty, no segments have beem found.")]
+    #[error("Video seems to be empty, no segments have been found.")]
```

Comment:
nit: we have nothing called "segments" anymore.

```suggestion
    #[error("Video is empty.")]
```